package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@unleash/proxy",
-  "version": "0.18.0",
+  "version": "0.18.2",
```

Comment:
Oh! Any idea what happened to 0.18.1?

Reply:
I created incorrect tag 0.18.1 pointing to 0.18.0 without bumping package.json. I prefer not to mutate anything and roll forward instead.

Reply:
Gotcha! No worries, was just curious